XPNet.GraphicsTest/GraphicsTestPlugin.cs

```diff
@@ -0,0 +1,144 @@
+﻿using XPNet;
+using System;
+using System.Numerics;
+//using OpenTK
+using XPNet.CLR;
+
+namespace XPNet
+{
+	/// <summary>
+	/// This is a testbed for getting the graphics going.
+	/// </summary>
+	[XPlanePlugin(
+		name: "Graphicstest",
+		signature: "mb.plugins.graphics",
+		description: "This should test the graphics."
+	)]
+
+	public class GraphicsTestPlugin : IXPlanePlugin
+	{
+		private readonly IXPlaneApi m_api;
+		private readonly IXPProbe m_probe;
+		private readonly IXPDrawingLoopHook drawingLoopHook;
+		private readonly IXPSceneryObject myPrivateBoeing;
+		private double incYaw;
+
+
+		public GraphicsTestPlugin(IXPlaneApi api)
+		{
+			m_api = api ?? throw new ArgumentNullException("api");
+
+			m_api.Log.Log("Displaytest started");
+			drawingLoopHook = m_api.Display.RegisterDrawCallback(DoSomething, XPLMDrawingPhase.xplm_Phase_Airplanes, 0);
+			m_api.Log.Log("And now create a probe");
+			m_probe = m_api.Scenery.CreateProbe();
+			m_api.Log.Log("Probe created");
+			m_api.Log.Log("Loading my private Boeing 737-800");
+			myPrivateBoeing = m_api.Scenery.LoadObject(@"Resources/CSL/BB_Boeing/B738/B738_NAX.obj");
```

Comment:
Anything that is needed in order to run tests should be included in the repository, so that anyone can run the test, not just you.  Don't include anything that you don't have full rights to contribute, however, so if that Boeing is a modified version of someone else's work, we'll need to load some other object, created entirely from scratch, for the test.  (Can't commit to the repository someone else's modified Boeing, or even a modified Boeing from X-Plane itself, without violating copyright.  If you made that Boeing entirely from scratch, and wanted to contribute it to the project, then you could include it ... but that seems unlikely, and we also wouldn't want to include any .obj file that was really _big_ in the project when a small test .obj would do, since everyone who gets the source would get a copy of the test object).

Reply:
Using now a tug from the standard scenery library, for this I wrapped also xplmlookupobjects.